pages/belt_docs/list.mdx

```diff
@@ -17,64 +17,158 @@ type t('a) = list('a);
 let length: t('a) => int;
 ```
 
+```reason
```

Comment:
just so you know, you can also use `re` for brewity :-)

Reply:
No way! Thanks!